packages/renderer/src/lib/dashboard/ProviderStatusTestHelper.ts

```diff
@@ -0,0 +1,72 @@
+/**********************************************************************
```

Comment:
I have only one remark. Here I think it's the first time we have a file only dedicated to the test but not with .spec.ts pattern, so are we sure it will not go to the final assembly/be excluded ?

Should we name it ProviderStatusTestHelper.spec.ts ?

Reply:
(I think it won't work with a spec.ts file but I don't know what should be the best approach like separate folder or another thing)